bundle/libraries/upload.go

```diff
@@ -111,6 +111,14 @@ func collectLocalLibraries(b *bundle.Bundle) (map[string][]configLocation, error
 				return v, fmt.Errorf("expected string, got %s", v.Kind())
 			}
 
+			sv, ok = file.GetByString("patched")
+			if ok {
+				patched, ok := sv.AsString()
+				if ok && patched != "" {
+					source = patched
+				}
+			}
```

Comment:
Since `sv` is not used outside of the `ok` conditional, we should inline the assignment in the conditional:
```go
if sv, ok = file.GetByString("patched"); ok {
```

I was alerted by the aliasing of `sv` for a second.

Reply:
thanks, fixed.